lib/kms_client.rb

```diff
@@ -3,7 +3,11 @@
 
 class KmsClient
   def initialize
-    @kms = Aws::KMS::Client.new(region: 'us-east-1')
+    @kms = Aws::KMS::Client.new(
+      region: 'us-east-1',
+      access_key_id:  ENV['AWS_ACCESS_KEY_ID'],
+      secret_access_key: ENV['AWS_SECRET_ACCESS_KEY']
+    )
```

Comment:
Was this change a deliberate choice, or did you find that it was required?

Reply:
Typically, the aws creds from the `~/.aws/credentials` file are used implicitly when the app runs. I found that in this docker setup, that wasn't true and I had to pass them in explicitly.